src/promptflow-evals/promptflow/evals/evaluators/_eci/_eci.py

```diff
@@ -65,8 +65,8 @@ class ECIEvaluator:
     .. code-block:: python
 
         {
-            "label": "False",
-            "reasoning": "Some reason."
+            "ECI_score": "False",
+            "ECI_reasoning": "Some reason."
```

Comment:
Should this be camel case ?

Reply:
I don't believe so, extant evaluators already return this results with snake-cased keys, this is just following that pattern.